firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -270,6 +259,11 @@ public static FirebaseFirestore testFirestore(
             new EmptyCredentialsProvider(),
             asyncQueue,
             /*firebaseApp=*/ null);
+    try {
+      AccessHelper.clearPersistence(firestore);
+    } catch (Exception e) {
```

Comment:
This catch clause is over broad and should be avoided unless it's required. See http://go/java-practices/exceptions#catch_exception.

Meanwhile note that any stack trace that comes from clearPersistence failing is going to include it so there isn't much value in catching this exception only to (indirectly) rethrow. It's less code to just `waitFor(clearPersistence(firestore));` and it will be just as obvious what's failing if it doesn't work.

I realize you're following the pattern of the AsyncQueue constructor above, but it's unwarranted there as well. Feel free to remove that try/catch block also.